cluster/helm/splice-validator/templates/validator.yaml

```diff
@@ -105,11 +105,28 @@ spec:
           value: |
             canton.validator-apps.validator_backend.scan-client.type = "trust-single"
             canton.validator-apps.validator_backend.scan-client.url = ${_scan.admin-api.address}
-        {{ else -}}
+        {{- else -}}
         - name: ADDITIONAL_CONFIG_BFT_SCAN
           value: |
+            {{- $config := .Values.scanClientConfig }}
+            {{- if eq ($config.useLastKnownConnectionsForInitialization | default true) false }}
+            canton.validator-apps.validator_backend.scan-client.use-last-known-connections-for-initialization = false
+            {{- end }}
+            {{- $hasThreshold := $config.threshold }}
+            {{- $hasTrustedSvs := $config.trustedSvs }}
+            {{- if and $hasThreshold $hasTrustedSvs (ge (len $hasTrustedSvs) ($hasThreshold | int)) }}
+            canton.validator-apps.validator_backend.scan-client.type = "bft-custom"
+            canton.validator-apps.validator_backend.scan-client.seed-urls = [ ${_scan.admin-api.address} ]
```

Comment:
this is actually fairly confusing, do you see a reason why not to use scanAddress directly?